adal/src/androidTest/java/com/microsoft/aad/adal/TokenCacheAccessorTests.java

```diff
@@ -128,6 +168,7 @@ static String createRawClientInfo(final String uid, final String utid) {
         MOCK_ID_TOKEN_WITH_CLAIMS = idTokenWithClaims;
     }
 
+    Context mContext;
```

Comment:
Is there a reason why mContext can't be final?

Reply:
It is not initialized inside of a constructor (but rather, inside of the test class' `setUp()` function)